app/Http/Controllers/JobofferController.php

```diff
@@ -105,6 +104,7 @@ public function update(Request $request, $id)
 
         $joboffer->title = $request->title;
         $joboffer->description = $request->description;
+        $joboffer->redirect_url = $request->redirect_url;
```

Comment:
Make sure this field and the description field are set to null when an empty string is given